lib/LaTeXML/Package/beamer_support_preload.sty.ltxml

```diff
@@ -0,0 +1,482 @@
+# /=====================================================================\ #
+# |  beamer_support_preload.sty                                         | #
+# | Implementation for LaTeXML                                          | #
+# |=====================================================================| #
+# | Part of LaTeXML:                                                    | #
+# |  Public domain software, produced as part of work done by the       | #
+# |  United States Government & not subject to copyright in the US.     | #
+# |---------------------------------------------------------------------| #
+# | Bruce Miller <bruce.miller@nist.gov>                        #_#     | #
+# | http://dlmf.nist.gov/LaTeXML/                              (o o)    | #
+# \=========================================================ooo==U==ooo=/ #
+package LaTeXML::Package::Pool;
+use strict;
+use warnings;
+use LaTeXML::Package;
+
+# This package is virtual and contains support code for beamer.
+# It's loaded by  beamer{article,}.cls *before* options are read.
+
+sub beamerTODO {
+  my ($feature) = @_;
+  Warn('todo', $feature, "beamer feature $feature is not yet supported; binding requires more development"); }
+
+#**********************************************************************
+# Custom Beamer Arguments
+#**********************************************************************
+
+# The 'Angled' (and corresponding 'OptionalAngled') parameter types are arguments in angle brackets.
+# The 'Angled' argument type is a literal argument type, meaning is not digested by default.
+# Example: <value>
+DefParameterType('Angled',         \&readAngled, reversion => \&revertAngled);
```

Comment:
The names are rather mysterious. Is `Angled` used for anything other than a Beamer action or alert or overlay?  Probably would be better to use something like `BeamerOverlay` as the type name.  You could possibly build the parsing into the type, as well, (if the timing is right), and avoid a lot of boilerplate.

Did you try `[Angled]` instead of `SquareAngled`? Seems like that should work, without a new name.

Reply:
I originally had is named `BeamerOverlay` but that sounded a bit too verbose for my taste.

I hadn't tried `[Angled]`, will check if that works.

Reply:
I have renamed `Angled` to the clearer `BeamerAngled`. 

> I've replaced SquareAngled by [Angled] as suggested in https://github.com/brucemiller/LaTeXML/commit/b114fa69dafdbdd9da6728e3cc0fe148d0c4ed97.

Turns out that doesn't actually work, there are subtle differences when a `[BeamerAngled]` is followed by another argument with square brackets (And I managed to re-generate one of the test cases and not even notice that it was horribly broken).
So I've had to re-add the argument type in 6051b578f7d58472eed53a67192b1a58329dbe99.